kratos/modeler/create_entities_from_geometries_modeler.cpp

```diff
@@ -0,0 +1,168 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ruben Zorrilla
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/parallel_environment.h"
+#include "utilities/entities_utilities.h"
+#include "utilities/variable_utils.h"
+#include "create_entities_from_geometries_modeler.h"
+
+namespace Kratos
+{
+
+namespace {
+
+/**
+ * @brief Create entities from geometries.
+ * @param EntityName the name of the entity
```

Comment:
This is not a name, its a `EntityIdentifier<T>`

Reply:
Done.